### DISCUSSION TITLE
登陆失败：密码错误或账号被冻结

### DISCUSSION THREAD

**Discussion by 5kind** [Mrs4s/go-cqhttp#1941] [Q&A]
今天在运行机器人时，不幸被风控，两号都同时被橄榄了，解封后尝试登陆时无效，显示标题所示报错：
**登陆失败：密码错误或账号被冻结**
并且再尝试登陆时，发现似乎无法使用 Android watch 以外的协议登陆，会报上述错误，账号仍可使用官方客户端群聊私聊，但无法扫码登陆机器人
在以手表协议登陆后，有一个账号无法正常接受所有消息（可能与添加好友群聊过多有关，不确定），私聊似乎没有问题，但是会忽略群聊聊天，同样是手表协议的另一账号就可以正常应答

**Comment by mxcoras** [comment]
亲测手机同时登了之后把没收到消息的群一个一个点进去，就能收到消息了，比较迷

**Reply by dannis999** [reply]
我之前也遇到了相同的情况，推测可能和消息免打扰有关，在手机上把群消息设置修改为接收并提醒就好了（我原来设置的是接收但不提醒）

**Comment by tp1415926535** [comment]
同样的情况，手机qq都自动退出登录了。
开始时扫码：”**登陆失败：密码错误或账号被冻结。**“
删除device.json之后，重新扫码：“**登录失败: 你的用户身份已失效，为保证帐号安全，请你重新登录。**“
请问怎么样才能继续登录？

**Reply by 5kind** [reply]
安卓手表协议是"protocol": 2，你这么改再登录试试

**Reply by tp1415926535** [reply]
> 安卓手表协议是"protocol": 2，你这么改再登录试试

确实可以了非常感谢

**Reply by zch11230** [reply]
我换成2也登成功了

**Comment by Jadlokin-Scarlet** [comment]
虽然可以换手表协议，但是文档说频道只支持Android Phone 和 iPad 请问除了更换手表协议以外有其他办法吗。
ps 我有同时使用了其他框架，那边提供了临时的解决方案可以密码登陆Android Phone 协议，好像是更新版本信息

**Reply by Jadlokin-Scarlet** [reply]
哦哦，尝试了最新的dev，登陆成功了
